Mathlib/LinearAlgebra/Charpoly/Univ.lean

```diff
@@ -0,0 +1,107 @@
+/-
+Copyright (c) 2024 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+import Mathlib.Data.MvPolynomial.Equiv
+import Mathlib.Data.MvPolynomial.Funext
+import Mathlib.LinearAlgebra.Matrix.Charpoly.Coeff
+import Mathlib.RingTheory.MvPolynomial.Homogeneous
+
+/-!
+# The universal characteristic polynomial
+
+In this file we define the universal characteristic polynomial.
```

Comment:
Can you add a reference (or a very short math explanation) about the universal characteristic polynomial? It doesn't seem (at least to me) completely standard terminology, but maybe it is just my fault.

Reply:
I added some docs.